### PR TITLE
Project Setup for Calculator App

### PR DESCRIPTION
**Pull request by ZawadiSumba66**
In this project I :
- Created a new react app.
- Installed big.js and prop-types
- Setup linters files

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ZawadiSumba66 👋

**You've done pretty well here. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "arrow-parens": ["error", "as-needed"]
+  },
```
Comment (mikenath223) [inline .eslintrc.json]: - [ ] Please ensure you use the exact rules stated [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json). It keeps your project professional as the rules are shared by other students also.

**Comment by mikenath223** [comment]
- [ ]  Also, in the readme ensure you add a Project title, description, and getting started instructions following the prescribed [readme template](https://github.com/microverseinc/readme-template).

**Review by whiz25 — APPROVED** [review]
## Approved 👏🏿 👏🏿 👏🏿 
Nice work making the requested changes. Readme and `.eslintrc.json` files updated as requested. Well done!


### Martin Afani
#### Slack: whiz25

**Pull request merged**